### PR TITLE
#214: don't show beta warning slide on return visit

### PR DESCRIPTION
**Pull request by tessaSAC**
For #214 

Tried so many iterations of this in both the `index` and `feedback-slide` components and still can't figure out why the `*ngIf` doesn't work even if hard-coding conditional to always be true. Halp pls 😰😰😰

### CODE REVIEW

**Comment by kirjs** [comment]
show me tonight

**Comment by tessaSAC** [comment]
Sounds good. Now on [another branch](https://github.com/AngularNYC/angular-presentation/tree/touredBefore) (similar fix) _as well as master_ can't even see the tour either. ◉◉💦
Today is just not my day re: GitHub I guess 😂😂😭

**Comment by tessaSAC** [comment]
So the logic works on the text of the Index slide, but I can't get ngIf to work at all on the feedback slide regardless of whether it's set to true or false. Have to go to bed soon but it really bothers me that it doesn't work -- feels like I must be missing something basic 🤔😰😩

**Comment by kirjs** [comment]
hm, I just tested it, and the slide is hidden?

**Comment by tessaSAC** [comment]
Yeah, no matter what I do I can't make it show, even with `*ngIf="1 === 1"` for example. I tested it on a header and it worked on that, I don't know why it doesn't work on the slide component 😨

File: src/app/demo/index/index.component.html [hunk 1]
```diff
@@ -4,20 +4,26 @@
     <slides-feedback-widget></slides-feedback-widget>
   </div>
 
-  <slides-slide id="index">
-    <h1>Angular Codelab</h1>
-    <h2>Welcome! Choose a topic below to start learning:</h2>
+  <div [ngSwitch]="firstVisit">
```
Comment (prestonvanloon) [inline src/app/demo/index/index.component.html]: ERROR in ng:///home/travis/build/AngularNYC/angular-presentation/src/app/demo/index/index.component.html (5,5): Property 'firstVisit' is private and only accessible within class 'IndexComponent'.

Comment (prestonvanloon) [inline src/app/demo/index/index.component.html]: Remove `private` in component.

File: src/app/presentation/feedback-slide/feedback-slide.component.ts [hunk 2]
```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
 
 export class FeedbackSlideComponent implements OnInit {
 
+  // firstVisit: boolean = true;
```
Comment (prestonvanloon) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Remove commented code

File: src/app/presentation/feedback-slide/feedback-slide.component.ts [hunk 3]
```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
...
+
   constructor(private presentation: PresentationComponent, private slide: SlideComponent) {
+    //  this.firstVisit = true;
```
Comment (prestonvanloon) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Remove commented code

File: src/app/presentation/feedback-slide/feedback-slide.component.ts [hunk 4]
```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
...
 
   closeSlide() {
-    this.presentation.nextSlide();
+    // this.presentation.nextSlide();
   }
 
   @HostListener('document:keypress', ['$event'])
   handleKeyboardEvent(event: KeyboardEvent) {
     if (this.slide.active) {
+      // this.firstVisit = false;
```
Comment (prestonvanloon) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Remove commented code

File: src/app/presentation/feedback-slide/feedback-slide.component.ts [hunk 5]
```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
...
+    //  this.firstVisit = true;
   }
 
   ngOnInit() {
+  localStorage.setItem('visitedBefore', 'yes');
   }
 
   closeSlide() {
-    this.presentation.nextSlide();
+    // this.presentation.nextSlide();
```
Comment (prestonvanloon) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Was this intentional?

File: src/app/presentation/feedback-slide/feedback-slide.component.ts [hunk 6]
```diff
@@ -10,19 +10,24 @@ import {SlideComponent} from '../slide/slide.component';
...
 export class FeedbackSlideComponent implements OnInit {
 
+  // firstVisit: boolean = true;
+
   constructor(private presentation: PresentationComponent, private slide: SlideComponent) {
+    //  this.firstVisit = true;
   }
 
   ngOnInit() {
+  localStorage.setItem('visitedBefore', 'yes');
```
Comment (prestonvanloon) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Indentation

**Comment by kirjs** [comment]
Let's chat about this slide in general, we'll need it back soon

**Comment by tessaSAC** [comment]
Cool, I'm guessing/hoping implementation might change a bit anyway with the new slide display workflow.

File: src/app/presentation/feedback-slide/feedback-slide.component.ts (hunk 5 above)
Comment (tessaSAC) [inline src/app/presentation/feedback-slide/feedback-slide.component.ts]: Sorry, probably not; this PR isn't commit-ready yet; still haven't figured out how to get the one-time display to work and still be able to navigate to other slides.

**Comment by kirjs** [comment]
I'm a bit lost across all the PRs. Is this one still needed?

**Comment by prestonvanloon** [comment]
@tessaSAC Please update this PR

**Comment by tessaSAC** [comment]
@prestonvanloon will be working on it today!
@kirjs it's the one that shows the "feedback please" intro slide -- do we still need it?

**Comment by tessaSAC** [comment]
No longer need feedback request slide

**Pull request closed without merging**